src/main/java/gov/cabinetofice/gapuserservice/dto/ValidateSessionsRolesRequestBodyDto.java

```diff
@@ -0,0 +1,13 @@
+package gov.cabinetofice.gapuserservice.dto;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Data
```

Comment:
Data gives getter and setters

Reply:
nice

Reply:
Would suggest taking advantage of records going forward